packages/components/psammead-story-promo/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 0.3.0-alpha.1 | [PR#522](https://github.com/bbc/psammead/pull/522) Add MediaIndicator |
```

Comment:
```suggestion
| 0.2.0-alpha.2 | [PR#522](https://github.com/bbc/psammead/pull/522) Add MediaIndicator |
```
Since `0.2.0` is not a stable version yet

Reply:
fixed! 😄